Source/WebCore/rendering/RenderMarquee.cpp

```diff
@@ -268,7 +268,8 @@ void RenderMarquee::timerFired()
         return;
     }
     
-    const RenderStyle& style = m_layer->renderer().style();
+    const CheckedRef style = m_layer->renderer().style();
```

Comment:
We don't normally mark local variables as `const.

Reply:
Fixed